components/02-elements/password-strength/password-strength.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="password-strength {{ class }}">
-    <div class="password-strength__progress {{ strengthProgressClass }}">
+    <div>
```

Comment:
You can remove this empty `div`

Reply:
no, I can't. I would like to keep the same structure like in Magento alpaca theme

Reply:
Ok